### PR TITLE
Change IE to Edge in dart2js.md

### PR DESCRIPTION
**Pull request by Aayush-B**
Making changes to dart2js.md file by replacing Internet Explorer with Edge. This addresses issue #2382

@kevmoo @kwalrath

### CODE REVIEW

**Comment by Aayush-B** [comment]
@googlebot I signed it!

File: src/tools/dart2js.md [hunk 1]
```diff
@@ -197,11 +197,11 @@ To debug in Chrome:
...
 
-### Internet Explorer {#dart2js-debugging-ie}
+### Edge {#dart2js-debugging-ie}
 
-To debug in Internet Explorer:
+To debug in Edge:
 
-1. Update to the latest version of Internet Explorer. (Source-map support
+1. Update to the latest version of Edge. (Source-map support
    was added to IE in April 2014).
```
Comment (kwalrath) [inline src/tools/dart2js.md:205]:
```suggestion
1. Update to the latest version of Edge.
```

File: src/tools/dart2js.md [hunk 2]
```diff
@@ -197,11 +197,11 @@ To debug in Chrome:
...
 2. Load **Developer Tools** (**F12**). For more information, see
    [Using the F12 developer tools.](https://msdn.microsoft.com/library/ie/bg182326(v=vs.85))
```
Comment (kwalrath) [inline src/tools/dart2js.md:207]: I think the URL needs updating. Maybe this one?

```suggestion
   [Using the F12 developer tools.](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide)
```

Comment (kwalrath) [reply src/tools/dart2js.md:207]: Should we delete the 4th item? I don't see Ctrl+Shift+E or "exception" in that page.

File: src/tools/dart2js.md [hunk 3]
```diff
@@ -197,11 +197,11 @@ To debug in Chrome:
...
 4. Reload your app.
 
-### Internet Explorer {#dart2js-debugging-ie}
+### Edge {#dart2js-debugging-ie}
 
-To debug in Internet Explorer:
+To debug in Edge:
 
-1. Update to the latest version of Internet Explorer. (Source-map support
+1. Update to the latest version of Edge. (Source-map support
```
Comment (kevmoo) [inline src/tools/dart2js.md:204]: Can get rid of the bit about source-map support.

File: src/tools/dart2js.md (hunk 2 above)
Comment (Aayush-B) [reply src/tools/dart2js.md:207]: Hi, we can keep the 4th item. You can find the Ctrl+Shift+E on this page : https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide/debugger

File: src/tools/dart2js.md (hunk 1 above)
Comment (Aayush-B) [reply src/tools/dart2js.md]: Made the changes and committed.

**Pull request merged**